install_files/ansible-base/roles/restrict-direct-access/tasks/iptables.yml

```diff
@@ -35,6 +47,8 @@
     owner: root
     group: root
     dest: /etc/network/iptables
+  when:
+    - ansible_distribution_release == 'xenial'
```

Comment:
If the `load_iptables` script is only copied under Xenial, then its content shouldn't change to address Focal support.

Reply:
Done.